### PR TITLE
[Update] Add TiSpark telemetry

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
TiSpark has telemetry in master branch and will have release the telemetry function in the next version. So we should add TiSpark telemetry instructions in telemtry.md.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9659
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Daemonxiao — APPROVED** [review]

File: telemetry.md [hunk 1]
```diff
@@ -60,6 +60,20 @@ To view the full content of the usage information shared to PingCAP, set the `TI
 TIUP_CLUSTER_DEBUG=enable tiup cluster list
 ```
 
+### TiSpark
+
+When the telemetry collection feature is enabled for TiSpark, the Spark module will share the usage details of TiSpark, including (but not limited to):
+
+- A randomly generated telemetry ID.
+- Some configuration information of TiSpark, such as the read engine and whether streaming read is enabled.
+- Cluster deployment characteristics, such as the machine hardware information, OS information, and component version number of the node where TiSpark is located.
```
Comment (TomShawn) [inline telemetry.md:69]:
```suggestion
- Cluster deployment information, such as the machine hardware information, OS information, and component version number of the node where TiSpark is located.
```

File: telemetry.md [hunk 2]
```diff
@@ -60,6 +60,20 @@ To view the full content of the usage information shared to PingCAP, set the `TI
...
+
+You can view TiSpark usage information that is collected through Spark logs. You can set the Spark log level to INFO or lower, for example:
```
Comment (TomShawn) [inline telemetry.md:71]:
```suggestion
You can view TiSpark usage information that is collected in Spark logs. You can set the Spark log level to INFO or lower, for example:
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**